cernopendata/modules/fixtures/data/articles/about-cms/cms.md

```diff
@@ -36,7 +42,8 @@ The following are provided through this portal:
     * Definition of same objects is different in each analysis.
 
 * The files can be read in [ROOT](http://root.cern.ch/), but they cannot be opened (and understood) as simple data tables.
-* Only the runs that are validated by data quality monitoring should be used in any analysis. The [list of the validated runs](/collection/CMS-Validation-Utilities) is provided.
+* Only the runs that are validated by data quality monitoring should be used in any analysis. The [list of the validated runs](/search?page=1&size=20&q="CMS list of validated runs") (FIXME markdown does not accpet hyphen in links?) is provided.
+* A small sample of [RAW data](/search?page=1&size=20&q=&experiment=CMS&file_type=raw) is also provided.
```

Comment:
Should this be "raw" rather than "RAW"?

Reply:
We are sort of used to RAW in CMS (i.e. RAW, RECO, AOD), but indeed here probably "raw data" is better